test/tests/parallel/set3/mta/dsn.bats

```diff
@@ -47,9 +47,9 @@ function teardown_file() {
 @test "should always send a DSN when requested" {
   export CONTAINER_NAME=${CONTAINER1_NAME}
 
-  _send_email 'email-templates/dsn-unauthenticated'
-  _send_email 'email-templates/dsn-authenticated' '0.0.0.0 465'
-  _send_email 'email-templates/dsn-authenticated' '0.0.0.0 587'
+  _nc_wrapper 'emails/nc_raw/dsn/unauthenticated.txt'
```

Comment:
Context:
- `swaks` hasn't had a new release in years, but has had recent activity with a release planned for Jan 2024 that may land the notify feature request to support dropping the `nc` approach.

Reply:
I think I added a comment in the GitHub repository, more precisely, in the tracking issue.

Ah, and I just noticed you've been there as well :D 

Here is the link for reference: <https://github.com/jetmore/swaks/issues/41>